window.go

```diff
@@ -531,3 +532,14 @@ func (w *Window) Unmaximize() (err error) {
 	_, err = synchronousEvent(w.ctx, w, w.w, Event{Name: EventNameWindowCmdUnmaximize, TargetID: w.id}, EventNameWindowEventUnmaximize)
 	return
 }
+
+// UpdateCustomOptions unmaximize the window
+func (w *Window) UpdateCustomOptions(o WindowCustomOptions) (err error) {
+	if err = w.ctx.Err(); err != nil {
+		return
+	}
+	w.m.Lock()
+	w.o.Custom = &o
+	w.m.Unlock()
+	return w.w.write(Event{WindowOptions: w.o, Name: EventNameWindowCmdUpdateCustomOptions, TargetID: w.id})
```

Comment:
Why not use `synchronuousEvent`?

Reply:
when testing with synchronousEvent, the event becomes blocking and prevent subsequent calls from being called

Reply:
Mmm makes sense. But `synchronousEvent` should still be used. To fix the problem:

- JS should write an event `window.event.updated.custom.options` [after this line](https://github.com/asticode/astilectron/blob/master/index.js#L280) (check out [here](https://github.com/asticode/astilectron/blob/master/index.js#L119) to see how to use `write`)
- GO should use `synchronousEvent` and use the `windowEventUpdatedCustomOptions` event you created:

```go
synchronousEvent(..., ..., ..., Event{Name: EventNameWindowCmdUpdateCustomOptions, TargetID: w.id}, EventNameWindowEventUpdatedCustomOptions)
```


I've already merged the JS PR but you can open a new one.

Reply:
Understood! will open a new JS PR